ui/component/livestreamScheduledInfo/view.jsx

```diff
@@ -13,15 +13,16 @@ const CALC_TIME_INTERVAL_MS = 1000;
 type Props = {
   // -- redux --
   releaseTimeMs: number,
+  appLanguage: string,
 };
 
 export default function LivestreamScheduledInfo(props: Props) {
-  const { releaseTimeMs } = props;
+  const { releaseTimeMs, appLanguage } = props;
 
   const [startDateFromNow, setStartDateFromNow] = React.useState();
   const [inPast, setInPast] = React.useState();
 
-  const startDate = React.useMemo(() => moment(releaseTimeMs).format('MMMM Do, h:mm a'), [releaseTimeMs]);
+  const startDate = React.useMemo(() => moment(releaseTimeMs).locale(appLanguage).format('LLL'), [releaseTimeMs]);
```

Comment:
The dependency is technically incomplete, and the render would be incorrect if `appLanguage` changes in the background while this component is mounted, say from a background Pref sync.

<img width="669" alt="image" src="https://user-images.githubusercontent.com/64950861/224132166-94d5bacb-cde6-4d7b-9d29-8d3910e2472a.png">

Reply:
@keikari can you circle back in the places we use this to make sure it's covered?

Reply:
Made PR https://github.com/OdyseeTeam/odysee-frontend/pull/2583
Checked the previous changes with localizing dates, and this was the only place where it was memoized